hedera-node/src/main/java/com/hedera/services/contracts/operation/HederaOperationUtil.java

```diff
@@ -172,19 +172,15 @@ public static long newContractExpiryIn(MessageFrame frame) {
 		// if this is a delegate call activeContract should be the recipient address
 		// otherwise it should be the contract address
 		if (isDelegateCall) {
-			sigReqIsMet = sigsVerifier.hasActiveKeyOrNoReceiverSigReq(
+			sigReqIsMet = sigsVerifier.hasActiveKeyOrNoReceiverSigReq(true,
 					account.getAddress(),
 					frame.getRecipientAddress(),
-					frame.getContractAddress(),
-					frame.getRecipientAddress(),
-					updater.aliases());
+					updater.wrappedTrackingLedgers());
 		} else {
-			sigReqIsMet = sigsVerifier.hasActiveKeyOrNoReceiverSigReq(
+			sigReqIsMet = sigsVerifier.hasActiveKeyOrNoReceiverSigReq(false,
 					account.getAddress(),
-					frame.getRecipientAddress(),
-					frame.getContractAddress(),
 					frame.getContractAddress(),
-					updater.aliases());
+					updater.wrappedTrackingLedgers());
```

Comment:
Using the `wrappedTrackingLedgers()` is more expensive and doesn't add anything here; let's make the `trackingLedgers()` method [here](https://github.com/hashgraph/hedera-services/blob/master/hedera-node/src/main/java/com/hedera/services/store/contracts/AbstractLedgerWorldUpdater.java#L263) public and call that one instead. 😁